{
  try {
    mNotificationManager.notify(id,notification);
  }
 catch (  RuntimeException runtimeException) {
    CrashlyticsUtils.logException(runtimeException,CrashlyticsUtils.ExceptionType.SPECIFIC,AppLog.T.UTILS,"See issue #2858 / #3966");
    AppLog.d(T.POSTS,"See issue #2858 / #3966; notify failed with:" + runtimeException);
  }
}
